billy-core-jpa/src/main/java/com/premiumminds/billy/persistence/entities/jpa/JPACustomerEntity.java

```diff
@@ -146,6 +154,11 @@ public void setTaxRegistrationNumber(String number) {
         this.taxId = number;
     }
 
+    @Override
+    public void setTaxRegistrationNumberCountry(final String country) {
```

Comment:
The same solution for the previous comment should also be applied here, whatever the chosen direction will be. Here there is consistency with the input naming, but on the other hand there is inconsistency with the previous place, when both should be consistent.